src/html_section.js

```diff
@@ -34,6 +34,19 @@ var HTMLSectionBuilder = function(){
 	this.stack = [new HTMLSection()];
 };
 
+HTMLSectionBuilder.scopify = function(renderer) {
```

Comment:
I noticed there's already a helper that does this sort of: https://github.com/canjs/can-stache/blob/namedPartials/src/utils.js